packages/block-library/src/quote/v2/deprecated.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { omit } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { createBlock, parseWithAttributeSchema } from '@wordpress/blocks';
+import { RichText, useBlockProps } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import deprecationsForV1Block from '../deprecated';
+
+export const migrateToQuoteV2 = ( attributes ) => {
```

Comment:
Do you think it's worth unit testing this function like I did in the list block?

Reply:
Makes sense https://github.com/WordPress/gutenberg/pull/39844/commits/2bdddf11d77c5cf18a31183f003240c5a489d20d